chrome/locale/de/quicktext.dtd

```diff
@@ -1,10 +1,10 @@
 <!ENTITY quicktext.label "Quicktext">
 <!ENTITY quicktext.shortname.label "Quicktext">
 
-<!ENTITY quicktext.settings.title "Quicktext Einstellungen">
+<!ENTITY quicktext.settings.title "Quicktext-Einstellungen">
 <!ENTITY quicktext.settings.label "Einstellungen">
 <!ENTITY quicktext.file.label "Datei">
-<!ENTITY quicktext.goToHomepage.label "Zur Homepage">
+<!ENTITY quicktext.goToHomepage.label "Zur Webseite">
```

Comment:
Homepage ist wohl wirklich falsch, aber wenn dann Website oder?

Reply:
IMO it should be "Website" - see:
https://webworker.com/homepage-website-oder-doch-webseite/
https://www.textschoepfung.de/warum-eine-homepage-keine-website-ist-und-eine-website-keine-webseite/

Reply:
https://www.duden.de/rechtschreibung/Webseite

Reply:
https://www.duden.de/suchen/dudenonline/Website